bundles/org.openhab.binding.vdr/src/main/java/org/openhab/binding/vdr/internal/VDRHandler.java

```diff
@@ -96,14 +97,16 @@ public void initialize() {
                 con.openConnection();
                 updateStatus(ThingStatus.ONLINE);
                 updateProperties(con);
-            } catch (SVDRPException se) {
+            } catch (SVDRPConnectionException ce) {
                 // also update power channel when thing is offline when initializing
                 thing.getChannels().stream().map(c -> c.getUID()).filter(this::isLinked).forEach(channelUID -> {
                     if (VDRBindingConstants.CHANNEL_UID_POWER.equals(channelUID.getIdWithoutGroup())) {
                         updateState(channelUID, OnOffType.OFF);
                     }
                 });
-                updateStatus(ThingStatus.OFFLINE);
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.NONE, ce.getMessage());
+            } catch (SVDRPParseResponseException se) {
+                updateStatus(ThingStatus.UNKNOWN, ThingStatusDetail.COMMUNICATION_ERROR, se.getMessage());
```

Comment:
The UNKNOWN status has no status detail nor message. You could make this OFFLINE, too.

Reply:
Please see my comment below for the other issue.